app/src/main/res/values/strings.xml

```diff
@@ -213,6 +213,7 @@
     <string name="hospital_info_tutorial_state_overview">These symbols give you a quick overview over the states of your devices.</string>
     <string name="hospital_info_tutorial_asset_list">You can browse through a list of your colleagues or devices by expanding the respective section.</string>
     <string name="hospital_info_tutorial_filter">Use this field for filtering.</string>
+    <string name="hospital_info_overdue_devices">devices are scheduled for maintenance</string>
```

Comment:
maybe we should use a text that creates more of a sense of urgency?
something like "xy devices should be maintained/serviced immediately"

Reply:
Current wording was decided on due to user experience. More wording can be implemeted later, when the devices are serviced on a regular basis.